Config/config.php

```diff
@@ -38,4 +38,34 @@
     | Expressed in bytes
     */
     'max-total-size' => 1000000000,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Define which assets will be available through the asset manager
+    |--------------------------------------------------------------------------
+    | These assets are registered on the asset manager
+    */
+    'media-partial-assets' => [
+        // Styles
+        'asgard.css' => ['theme' => 'css/asgard.css'],
```

Comment:
Your right should leave this open so users doesn't have to create asgard.css in his theme.
It should be themes responsibility to load styles for module.